JournalReligionSociety.js

```diff
@@ -0,0 +1,177 @@
+{
+	"translatorID": "e969da6b-30c9-4b26-8fcc-c2d78bce685f",
+	"label": "JournalReligionSociety",
+	"creator": "Vincent Carret",
+	"target": "^https?://(www\\.)?moses.creighton.edu/JRS",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-11-21 19:29:09"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Vincent Carret
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	// Three possible cases : the list of articles of an issue, the list of content of a supplement, or the list of the supplements
+	if (getSearchResults(doc, true) || url.search('/toc/SS') > -1 || url.search('/toc/Supplement') > -1) {
```

Comment:
`getSearchResults` should be an & condition (and then the two OR conditions need to be in a parentheses -- if it doesn't return `true`, `doWeb` will fail.

Reply:
also, use `includes` for literal string matching